build.gradle.kts

```diff
@@ -138,9 +139,14 @@ subprojects.filter { listOf("roaringbitmap", "bsi").contains(it.name) }.forEach
 }
 
 
-tasks {
-    register("build") {
-        // dummy build task to appease release plugin
+subprojects {
+    apply(plugin = "com.diffplug.spotless")
+    spotless {
+        java {
+            googleJavaFormat("1.17.0").aosp().reflowLongStrings()
```

Comment:
I doubt you want `aosp()`, especially as it leads to 4-spaces indentation (https://github.com/diffplug/spotless/issues/420#issuecomment-512579445)